doc/nrf/libraries/bin/lwm2m_carrier/CHANGELOG.rst

```diff
@@ -9,15 +9,61 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
-liblwm2m_carrier 3.3.3
+Certification status
+====================
+
+For certification status of the released versions, see `Mobile network operator certifications`_.
+
+liblwm2m_carrier 3.4.0
 **********************
 
-Release for modem firmware version 1.3.5 and 2.0.0.
+Release for modem firmware version 1.3.5, 1.3.6, 2.0.0, and 2.0.1.
 
-Certification status
-====================
+Size
+====
 
-For certification status, see `Mobile network operator certifications`_.
+See :ref:`lwm2m_lib_size` for an explanation of the library size in different scenarios.
+
++-------------------------+---------------+------------+
+|                         | Flash (Bytes) | RAM (Bytes)|
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (binary)                |               |            |
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (reference application) |               |            |
++-------------------------+---------------+------------+
+
+Changes
+=======
+
+* Removed AT&T support.
+  The LwM2M carrier library is no longer required to certify with AT&T.
+
+* Added preliminary support for Bell Canada subscriber ID.
+  This carrier can be disabled or enabled with the Kconfig option :kconfig:option:`CONFIG_LWM2M_CARRIER_BELL_CA`.
+
+* Changed the default string of the Device Type resource to say "Module" instead of "Smart Device".
+  This can be changed to other strings using :kconfig:option:`CONFIG_LWM2M_CARRIER_DEVICE_TYPE`.
+
+* Added ``disable_queue_mode`` to the configuration :c:macro:`lwm2m_carrier_config_t`.
+  Queue Mode can now be disabled using the Kconfig :kconfig:option:`CONFIG_LWM2M_CARRIER_QUEUE_MODE`.
```

Comment:
```suggestion
  Queue Mode can now be disabled using the Kconfig option :kconfig:option:`CONFIG_LWM2M_CARRIER_QUEUE_MODE`.
```

Reply:
Will keep "Queue Mode" since that is the technical term, as opposed to "Queue" mode. (But will add Kconfig option)